### ISSUE TITLE
Error: The non-abstract class 'InputConnectionController' is missing implementations: TextInputClient.performPrivateCommand

### ISSUE THREAD

**Issue by Danielnator** [memspace/zefyr#427]
## Steps to Reproduce

I have recently updated flutter to version 1.22.2. I have forked the zefyr package, however I have done only a few UI changes and I have done the last pull request on October 5th where I had no merge conflict (hence I have the most up-to-date code of zefyr).

However, when building in debug mode, I get an error that a non-abstract class is missing an implementation. There are 4 suggested solutions, however I am not sure where and how to implement them. Did anyone have the same issue, or is this issue already worked on?

**Logs:**
`../../flutter/flutter/.pub-cache/git/zefyr-99068c529bb8824235ad0c4bddf5d81de854f9d5/packages/zefyr/lib/src/widgets/input.dart:11:7: Error: The non-abstract class 'InputConnectionController' is missing implementations for these members:
 - TextInputClient.performPrivateCommand
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class InputConnectionController implements TextInputClient {
      ^^^^^^^^^^^^^^^^^^^^^^^^^
../../flutter/flutter/packages/flutter/lib/src/services/text_input.dart:813:8: Context: 'TextInputClient.performPrivateCommand' is defined here.
  void performPrivateCommand(String action, Map<String, dynamic> data);
       ^^^^^^^^^^^^^^^^^^^^^

FAILURE: Build failed with an exception.`

**Flutter doctor**

```
[√] Flutter (Channel stable, 1.22.2, on Microsoft Windows [Version 10.0.18363.1139], locale de-DE)
[!] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses
[√] Android Studio (version 4.0)
[√] Connected device (1 available)


```

**Comment by Danielnator** [comment]
Sorry, this is a duplicate issue, hence I am closing it.

**Comment by Danielnator** [comment]
I am reopening, since the solutions presented in the previous posts are not helping me. I have the up-to-date code from github and I get the error anyway.

**Comment by pulyaevskiy** [comment]
I'm pretty sure that this should be addressed in 0.12.0 which is on Pub https://pub.dev/packages/zefyr. Have you tried it?

**Comment by Danielnator** [comment]
@pulyaevskiy You are right, it is fixed in the pub.dev package. I have pulled from the wrong branch. Thanks!
